platform/linux-generic/odp_ml.c

```diff
@@ -1096,6 +1100,8 @@ static const char *data_type_str(odp_ml_data_type_t data_type)
 		return "fp32";
 	case ODP_ML_DATA_TYPE_BFP16:
 		return "bfp16";
+	case ODP_ML_DATA_TYPE_FP64:
+		return "fp64";
```

Comment:
I'd perhaps put this below `ODP_ML_DATA_TYPE_FP32` for uniformity, but either way is OK for me.